sova/awareness/rendering/cli_renderer.py

```
@@ -0,0 +1,159 @@
+def _urgency_marker(urgency: int) -> str:
+    """Return urgency marker text + color."""
+    if urgency >= 2:
+        return "[red]!!![/red]"
+    if urgency == 1:
+        return "[yellow]!![/yellow]"
+    return "[dim]![/dim]"
+
+
+def _format_time_ago(timestamp: datetime) -> str:
+    """Format timestamp as relative time (e.g., '2h ago', '30m ago')."""
+    if timestamp > datetime.now():
```

Ruff (0.16.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)